src/copyFile.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable no-console */
+'use strict';
+
+const fs = require('fs');
+const { terminal } = require('./terminal');
+
+const copyFile = (source, dest) => {
+  if (source === dest) {
+    console.log(
+      '\x1b[43m\x1b[36m Please check the routing!!!\n'
+      + ' You are trying to copy a file to the same folder... \x1b[0m',
+    );
+
+    terminal.close();
+  }
+
+  if (source !== dest) {
+    fs.copyFile(source, dest, error => {
+      if (error) {
+        console.log(error);
```

Comment:
Use error levels to play correctly
```suggestion
        console.error(error);
```

Reply:
well noted! thank you! 👍